### ISSUE TITLE
Time log display problem

### ISSUE THREAD

**Issue by SaeidJavadi** [Busimus/cutelog#44]
![Screenshot 2024-04-24 115500](https://github.com/busimus/cutelog/assets/53658372/832d7187-4bbb-413f-8719-f7be10b2ace9)

When we import a log file to the program, there is a problem in the time table that does not show the events of each log correctly! And it shows the time of importing the file to the program and it is shown for all the logs of the same time! In case, the time of occurrence of the log must be the same as the time recorded in the file
As you can see in the photo, the login time is what is displayed at the bottom of the menu (the details section of each log) and the time in the table is all wrong.

**Comment by Intergalactyc** [comment]
Yeah, unfortunately looks like you have to name your timestamp "created" or "time", and it has to be a float POSIX timestamp in seconds (python logging record `record.created` attribute, or using the time module `time.time_ns()/1e9`, otherwise it won't work.

The relevant lines of code in Cutelog's source are
```python
self.created = logDict.get("created")
if self.created is None:
    self.created = logDict.get("time")
    if self.created is None:
        self.created = logDict.get("_created")
if self.created is None or type(self.created) not in (int, float):
    self.created = datetime.now().timestamp()
```
in `logger_tab.py`. I don't have the time to make a fix pr but it's a simple change if the dev or a contributor wants to make the timestamps more flexible for loaded (non-socketed) custom logs.
